molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml

```diff
@@ -205,14 +208,23 @@ provisioner:
 
         kerberos_principals:
           # Wish there was a better way to create this list, this will do for now
-          - principal: "${CONTROLLER_HOSTGROUP:-zookeeper}/${KRAFT_CONTROLLER:-zookeeper}1.confluent@{{kerberos.realm | upper}}"
-            keytab_path: "keytabs/${CONTROLLER_HOSTGROUP:-zookeeper}-${KRAFT_CONTROLLER:-zookeeper}1.keytab"
+          - principal: "zookeeper/zookeeper1.confluent@{{kerberos.realm | upper}}"
```

Comment:
It would be cleaner if you put a conditional check on CONTROLLER_HOSTGROUP variable nand populate the principal accordingly